packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -123,6 +128,15 @@ export default class DataGrid extends React.Component {
 		}
 	}
 
+	onBodyScroll(event) {
+		if (event.direction === AG_GRID.SCROLL_VERTICAL_DIRECTION) {
+			this.props.onVerticalScroll({
```

Comment:
please try has hard has you can to keep event, payload as API arg on event handler 🙏